src/samples/dolphin/dolphin.cpp

```diff
@@ -14,13 +14,15 @@
 const u8 WATER_TILE_SCALE = 5;
 const u8 WATER_SIZE = 100;
 
-const u8 OYSTERS_COUNT = 15;
+const u8 MINES_COUNT = 15;
+const u8 OYSTERS_COUNT = 5;
 
 float Dolphin::engineFPS = 60.F;
 
 Dolphin::Dolphin(Engine *t_engine) : engine(t_engine), camera(&engine->screen)
 {
     oysters = new Collectible[OYSTERS_COUNT];
+    mines = new Mine[MINES_COUNT];
 }
 
 Dolphin::~Dolphin() {}
```

Comment:
I know that we have there infinite loop, but this is a good practice to release memory here, after allocation in constructor :)